src/pages/limyujin/Main/Main.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import _ from 'lodash';
+import Feeds from '../Main/Components/Feeds';
+import TopNav from '../Main/Components/TopNav';
+import MainRight from '../Main/Components/MainRight';
+import './Main.scss';
+
+class Main extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      feedInfo: [],
+      recommendData: [],
+      preItems: 0,
+      items: 2,
+    };
+  }
+
+  componentDidMount() {
+    this.getFeedData();
+    this.getRecommendData();
+    window.addEventListener('scroll', _.throttle(this.infinityScroll, 500));
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('scroll', this.infinityScroll);
+  }
+
+  getFeedData = () => {
+    const { preItems, items, feedInfo } = this.state;
+    const FEED_DATA = 'Data/limyujin/feedData.json';
+    fetch(FEED_DATA)
+      .then(res => res.json())
+      .then(feedData => {
+        let sliceData = feedData.slice(preItems, items);
+        this.setState({ feedInfo: [...feedInfo, ...sliceData] });
+      });
+  };
+
+  getRecommendData = () => {
+    const RECOMMEND_DATA = 'Data/limyujin/recommendData.json';
+    fetch(RECOMMEND_DATA)
+      .then(res => res.json())
+      .then(recommendData => {
+        this.setState({ recommendData: recommendData });
```

Comment:
```suggestion
        this.setState({ recommendData });
```
key value 같을 경우에는 객체 단축속성명 활용하실 수 있습니다

Reply:
수정 완료했습니다!